bazel-mode.el

```diff
@@ -21,6 +21,31 @@
 ;;
 ;;; Code:
 
+(defcustom buildifier-cmd "buildifier"
+  "Filename of buildifier executable."
+  :type 'string
+  :group 'bazel-mode)
+
+(defun buildifier ()
+  "Format current buffer using buildifier."
+  (interactive)
+  (let ((build-file-contents (buffer-string))
+		(input-buffer (current-buffer))
+		(orig-point (point)))
+	(with-current-buffer (get-buffer-create "*buildifier*")
+	  (erase-buffer)
+	  (insert build-file-contents)
+	  (let ((return-code
+			 (call-process-region (point-min) (point-max) buildifier-cmd t t)))
```

Comment:
Can you make this work remotely using `process-file-region`?

Reply:
Can you clarify? process-file-region doesn't look to exist.